go.mod

```diff
@@ -20,57 +20,56 @@ require (
 	github.com/dgrijalva/jwt-go v3.2.0+incompatible
 	github.com/dgrijalva/jwt-go/v4 v4.0.0-preview1
 	github.com/dgryski/go-farm v0.0.0-20200201041132-a6ae2369ad13
-	github.com/dgryski/go-groupvarint v0.0.0-20190318181831-5ce5df8ca4e1
-	github.com/docker/docker v1.13.1
+	github.com/dgryski/go-groupvarint v0.0.0-20230630160417-2bfb7969fb3c
+	github.com/docker/docker v24.0.5+incompatible
```

Comment:
I believe this upgrade had some breaking changes; see this failure [here](https://github.com/dgraph-io/dgraph/actions/runs/5907467276/job/16025467433?pr=8963)

Reply:
Done.